Panic when streaming logs

```
 "posetonflow-staging"
panic: runtime error: index out of range

goroutine 1 [running]:
runtime.panic(0x2ada00, 0x63bed7)
    /opt/go-1.2.1/src/pkg/runtime/panic.c:266 +0xb6
appsdeck/apps.streamLogs(0x2107ab6c0, 0x5f, 0x22108ca901)
    /run/media/leo/0a408ca7-d449-4157-9b44-8692a40edd7b/Appsdeck/appsdeck/src/appsdeck/apps/logs.go:54 +0x2b8
appsdeck/apps.Logs(0x2107bddc0, 0x13, 0x1, 0x14, 0x0, ...)
    /run/media/leo/0a408ca7-d449-4157-9b44-8692a40edd7b/Appsdeck/appsdeck/src/appsdeck/apps/logs.go:41 +0x590
appsdeck/cmd.func·005(0x2107bdd00)
    /run/media/leo/0a408ca7-d449-4157-9b44-8692a40edd7b/Appsdeck/appsdeck/src/appsdeck/cmd/logs.go:27 +0x167
github.com/codegangsta/cli.Command.Run(0x314320, 0x4, 0x3135c0, 0x1, 0x366830, ...)
    /run/media/leo/0a408ca7-d449-4157-9b44-8692a40edd7b/Appsdeck/appsdeck/src/github.com/codegangsta/cli/command.go:73 +0x994
github.com/codegangsta/cli.(*App).Run(0x210802210, 0x210763000, 0x5, 0x5, 0x6, ...)
    /run/media/leo/0a408ca7-d449-4157-9b44-8692a40edd7b/Appsdeck/appsdeck/src/github.com/codegangsta/cli/app.go:111 +0x855
main.main()
    /run/media/leo/0a408ca7-d449-4157-9b44-8692a40edd7b/Appsdeck/appsdeck/src/appsdeck/main.go:29 +0x2c1

goroutine 3 [syscall]:
os/signal.loop()
    /opt/go-1.2.1/src/pkg/os/signal/signal_unix.go:21 +0x1e
created by os/signal.init·1
    /opt/go-1.2.1/src/pkg/os/signal/signal_unix.go:27 +0x31

goroutine 6 [finalizer wait]:
runtime.park(0xbe10, 0x63f7e0, 0x63dd48)
    /opt/go-1.2.1/src/pkg/runtime/proc.c:1342 +0x66
runfinq()
    /opt/go-1.2.1/src/pkg/runtime/mgc0.c:2279 +0x84
runtime.goexit()
    /opt/go-1.2.1/src/pkg/runtime/proc.c:1394

goroutine 7 [runnable]:
net/http.(*persistConn).readLoop(0x2107bf800)
    /opt/go-1.2.1/src/pkg/net/http/transport.go:778 +0x68f
created by net/http.(*Transport).dialConn
    /opt/go-1.2.1/src/pkg/net/http/transport.go:528 +0x607

goroutine 8 [select]:
net/http.(*persistConn).writeLoop(0x2107bf800)
    /opt/go-1.2.1/src/pkg/net/http/transport.go:791 +0x271
created by net/http.(*Transport).dialConn
    /opt/go-1.2.1/src/pkg/net/http/transport.go:529 +0x61e

goroutine 10 [chan receive]:
net/http.(*persistConn).readLoop(0x2107bf680)
    /opt/go-1.2.1/src/pkg/net/http/transport.go:778 +0x68f
created by net/http.(*Transport).dialConn
    /opt/go-1.2.1/src/pkg/net/http/transport.go:528 +0x607

goroutine 11 [select]:
net/http.(*persistConn).writeLoop(0x2107bf680)
    /opt/go-1.2.1/src/pkg/net/http/transport.go:791 +0x271
created by net/http.(*Transport).dialConn
    /opt/go-1.2.1/src/pkg/net/http/transport.go:529 +0x61e
```

--- a/src/appsdeck/apps/logs.go
+++ b/src/appsdeck/apps/logs.go
@@ -27,7 +27,6 @@ func Logs(appName string, stream bool, n int) error {
 	if err = json.NewDecoder(res.Body).Decode(&app); err != nil {
 		return err
 	}
-	fmt.Printf("%+v\n", app)
 
 	res, err = api.Logs(app.LogsURL, stream, n)
 	if err != nil {
@@ -49,6 +48,12 @@ func streamLogs(res *http.Response) error {
 	for line, _, err := reader.ReadLine(); err == nil; line, _, err = reader.ReadLine() {
 		if len(line) != 0 {
 			parsedLine := strings.SplitN(string(line), ":", 2)
+			if len(parsedLine) != 2 {
+				// Invalid content from server, SSE should be
+				// msgname: content
+				// Anything else is wrong
+				continue
+			}
 			fmt.Println(
 				html.UnescapeString(
 					strings.TrimSpace(parsedLine[1]),

